test/CMakeLists.txt

```diff
@@ -83,7 +83,8 @@ externalproject_add(inttest_bin
 	"-DGTEST_ROOT:path=${GTEST_ROOT}"
 	"-DGMOCK_ROOT:path=${GMOCK_ROOT}"
 	"-DZNC_BIN_DIR:path=${CMAKE_INSTALL_FULL_BINDIR}"
-	"-DQt5_HINTS:path=${brew_qt5}")
+	"-DQt5_HINTS:path=${brew_qt5}"
+	"-DWITH_OTR:bool=${OTR_FOUND}")
```

Comment:
That will make the test fail on cygwin/appveyor where non-ancient version of libotr is not available.